.env.development

```diff
@@ -1 +1,3 @@
 REACT_APP_FAQ_ENDPOINT="https://covidfaq.apps.dev.ca-central-1.dialoguecorp.com"
+REACT_APP_RASA_SOCKET_ENDPOINT="TBD"
```

Comment:
these should be configured, no?

Reply:
Yes, but the backend is not deployed yet. It should be soon, we should probably wait before merging this.